fundingmanager.go

```diff
@@ -1265,19 +1296,22 @@ func (f *fundingManager) waitForFundingWithTimeout(completeChan *channeldb.OpenC
 // when a channel has become active for lightning transactions.
 // The wait can be canceled by closing the cancelChan.
 func (f *fundingManager) waitForFundingConfirmation(completeChan *channeldb.OpenChannel,
-	cancelChan <-chan struct{}, doneChan chan<- struct{}) {
-
-	defer close(doneChan)
+	cancelChan <-chan struct{}, errChan chan<- error,
+	confirmChan chan<- *chainntnfs.TxConfirmation) {
 
 	// Register with the ChainNotifier for a notification once the funding
 	// transaction reaches `numConfs` confirmations.
 	txid := completeChan.FundingOutpoint.Hash
 	numConfs := uint32(completeChan.NumConfsRequired)
+	if (numConfs < 6) {
+		numConfs = 6
```

Comment:
This will be wrong. This will prevent the `fundingLocked` being sent with < 6 confirmations, which is not what we want.  This check should be done _after_ `fundingLocked` is sent.